config/nvblox/specializations/nvblox_isaac_sim.yaml

```diff
@@ -1,3 +0,0 @@
-/**:
-  ros__parameters:
```

Comment:
Why is this file deleted? Because it is still referenced in nvblox.launch.py, so I don't think it should be deleted.

Reply:
The referenced code was deleted